packages/core/src/document.ts

```diff
@@ -164,48 +238,12 @@ class Document extends EventEmitter {
    * @param version - Document version
    */
   async loadVersion<T extends Doctype>(version: CID): Promise<T> {
-    const doc = await Document.loadVersion<T>(this, version, this._validate)
+    const versionId = DocID.fromOther(this.id, version)
+    const opts: DocOpts = {anchor: false, publish: false, sync: false}
+    const doc = await Document.load<T>(versionId, this._doctypeHandler as DoctypeHandler<T>, this.dispatcher, this.pinStore, this._context, opts, this._validate)
```

Comment:
Do we want to keep this method? A bit confusing that there are still two  ways of loading a version of a doc?